src/main/java/com/softserve/academy/spaced/repetition/domain/User.java

```diff
@@ -83,4 +83,22 @@ public void setCourses(Set<Course> courses) {
         this.courses = courses;
     }
 
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
Missing curly brackets here and on the next line.

Reply:
yes, I'll fix it